src/stories/EReceipt.stories.tsx

```diff
@@ -1,8 +1,16 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
 /* eslint-disable rulesdir/prefer-actions-set-data */
```

Comment:
Will it work in one line?
```suggestion
/* eslint-disable @typescript-eslint/naming-convention, rulesdir/prefer-actions-set-data */
```